shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -346,6 +346,15 @@ void TizenViewElementary::Show() {
   evas_object_show(event_layer_);
 }
 
+void TizenViewElementary::KeyEvent(const char* key,
+                                   const char* string,
+                                   const char* compose,
+                                   uint32_t modifiers,
+                                   uint32_t scan_code,
+                                   bool is_down) {
+  FT_UNIMPLEMENTED();
```

Comment:
Will this ever be implemented in the future or be no-op?

Reply:
Actually, I'm not sure if this will ever be used in `ElmTizenVIew`.
I modified it to `no-op`.

Reply:
We may have a better design in the future..

Reply:
I wrapped the code used only for nui with `NUI_SUPPORT` and deleted TizenViewElementary::OnKey.
Could you please review it again?

Reply:
I'm afraid the code is not very predictable but I'm okay with the change.